fe/src/pages/user/Profile/Index/ProfileIndex.jsx

```diff
@@ -30,91 +37,85 @@ function ProfileIndex() {
   return (
     <Fragment>
       <ToastContainer />
-      <Header />
-      <div className="container card p-2">
-        <div className="row gx-2">
-          <div className="col-lg-4 col-md-4 mb-2">
-            <div className="p-3 card bg-light">
-              <div className="d-flex justify-content-center">
-                <img
-                  className="mx-auto rounded-circle"
-                  style={{ width: 200, height: 200 }}
-                  src={currentUser.avatar ?? "/images/default_image.jpg"}
-                  alt="avatar"
+
+      <Container>
+        <Card style={"p-2"}>
+          <GridRow style={"gx-2"}>
+            <GridColumn style={"col-lg-4 col-md-4 mb-2"}>
+              <Card style={"p-3 bg-light"}>
+                <Avatar
+                  img={currentUser.avatar}
+                  customStyle={{ width: 200, height: 200 }}
                 />
-              </div>
-            </div>
-            <div className="mt-2 d-flex justify-content-center">
-              <p className="fw-bold">
-                {currentUser.first_name?.toUpperCase()}{" "}
-                {currentUser.last_name?.toUpperCase()}
-              </p>
-            </div>
+              </Card>
 
-            {currentUser.id !== parseInt(localStorage.getItem("user_id")) && (
-              <Fragment>
-                <div className="row mt-1 mb-4 w-75 mx-auto">
-                  <Followers />
-                </div>
+              <LayoutCenterChildren>
+                <p className="fw-bold">
+                  {currentUser.first_name?.toUpperCase()}{" "}
+                  {currentUser.last_name?.toUpperCase()}
+                </p>
+              </LayoutCenterChildren>
 
-                <div className="mb-2 d-flex justify-content-center">
-                  <hr className="w-50" />
-                </div>
-              </Fragment>
-            )}
+              {currentUser.id !== getUserId() && (
```

Comment:
- create a computed props in your `useProfile` hook. so you don't need to have these lengthy conditions
```
const isCurrentUser = currentUser.id === getUserId();
const isGuest = currentUser.id !== getUserId();
```

- import it in the page
```suggestion
              {isGuest && (
```